ideasbox/library/forms.py

```diff
@@ -4,15 +4,24 @@
 
 from .models import Book, BookSpecimen
 from .utils import fetch_from_openlibrary, load_from_moccam_csv, load_unimarc
+from  django.core.exceptions import ValidationError
 
 
 class BookSpecimenForm(forms.ModelForm):
+    
+    def clean_spfile(self):
+        # Does not allow to enter a file and a serial
+        if (self.cleaned_data['spfile']!=None)and(self.cleaned_data.get('serial')!=''):
```

Comment:
Not [pep8](https://www.python.org/dev/peps/pep-0008/) compliant. You should find a plugin for your favorite editor that warn you about that (look for flake8 plugin).